content/travisci/datachecking/_index.md

```diff
@@ -1,18 +1,18 @@
 ---
 date: 2016-04-09T16:50:16+02:00
-title: Add data checks
+title: Add Data Checks
 weight: 8
 pre: "<b>8. </b>"
 ---
 
-1. Modify the .R scripts in the `testthat` folder to automatically check your data for errors. For example, copy [this script](/sample-scripts/test-periods.R) into `testthat/test-periods.R`. It will automatically make sure that the sampling period values in your data are plausible.
+1. Modify the .R scripts in the `testthat` folder to automatically check your data for errors. GitHub Actions will automatically run any files inside the `testthat` folder. For example, copy [this script](/sample-scripts/test-periods.R) into `testthat/test-periods.R`. It will automatically make sure that the sampling period values in your data are plausible.
```

Comment:
This change is on the Travis CI page and probably needs to be moved to the GH Actions page.